content/en/blog/intro-development-falco.md

```diff
@@ -0,0 +1,222 @@
+
+
+#### Building Falco from scratch
+
+Most of the time, the main starting point for developing something is building it from scratch. You can fork `falco` and `libs`, and clone them into your development machine. Forking the repositories first is recommended if you want to later push the changes and save your work.
+
+Then, try to follow the steps in our [official Falco documentation](https://falco.org/docs/getting-started/source/). Some of the most useful `CMake` definitions you may want to use are:
+
+- `-DUSE_BUNDLED_DEPS=ON` allows you to download and compile all the needed Falco dependencies. This also helps to build Falco independently of the libraries installed in your system. <br>
+The build process will run trouble-free using this option, although it will be a little bit slower.
+
+- `-DFALCOSECURITY_LIBS_SOURCE_DIR=/path/to/local/libs` allows you to compile Falco with a local version of libs. If not specified, `CMake` will download the `libs` repository directly from GitHub. <br>
+This option is extremely handy when you are implementing something that requires modification to both `falco` and `libs` repositories so that you can easily test what you coded.
+
+- `-DBUILD_BPF=ON` allows compilation of the eBPF probe, crucial when developing something on the eBPF driver.
+
+- `-DCMAKE_BUILD_TYPE=Debug` is helpful when you want to debug something. Debug information is printed out and assertions are enabled.
+
+The complete sequence of commands you may want to run is:
+
+```shell
+$ mkdir build && cd build
+$ cmake -DUSE_BUNDLED_DEPS=ON \
+        -DFALCOSECURITY_LIBS_SOURCE_DIR=/path/to/local/libs \
+        -DBUILD_BPF=ON \
+        -DCMAKE_BUILD_TYPE=Debug  ..
+$ make
+```
+
+> Also, note that you can use the `-j` option of make to spawn multiple parallel jobs.
+
+![](https://falco.org/img/intro-development-falco/intro-development-falco-02.png)
+
+### Testing your build
+
+Now that you have successfully compiled Falco, try to run it. The default driver is the kernel module. To use it, assuming that you are in the build directory, just type:
+
+```shell
+$ sudo insmod driver/falco.ko
+$ sudo ./userspace/falco/falco -c ../falco.yaml \
+       -r ../rules/falco_rules.yaml
+```
+
+To start Falco with the eBPF driver instead, you need to set the `FALCO_BPF_PROBE` environment variable, like this:
+
+```shell
+$ sudo ./userspace/falco/falco FALCO_BPF_PROBE=driver/bpf/probe.o \
+       -c ../falco.yaml  -r ../rules/falco_rules.yaml
+```
+
+#### Falco Event-Generator
+
+Now that you built Falco, you may wonder if everything works as expected. A quick and dirty way of testing Falco is using the [event-generator](https://github.com/falcosecurity/event-generator).
+
+This is yet another project from the Falco security organization, and it can be used to generate some suspicious actions on the system, therefore, triggering some Falco rules.
```

Comment:
```suggestion
This is yet another project from the Falcosecurity organization, and it can be used to generate some suspicious actions on the system, therefore, triggering some Falco rules.
```